pygmt/tests/test_session_management.py

```diff
@@ -43,10 +44,12 @@ def test_gmt_compat_6_is_applied(capsys):
             )
             assert err == ""  # double check that there are no other errors
     finally:
-        with Session() as lib:
-            # revert gmt.conf back to GMT_COMPATIBILITY = 6
-            lib.call_module("set", "GMT_COMPATIBILITY 6")
```

Comment:
These lines are not needed, as they only change the gmt.conf in the session directory, which will be deleted after the `end()` statement.